spec/integration_spec.rb

```diff
@@ -299,44 +402,44 @@ def nb_children
 
 # create this index before the class actually loads, to ensure the rankingRules are updated
 # index = MeiliSearch::Index.new(safe_index_name('City_replica2'))
-index = MeiliSearch.client.create_index(safe_index_name('City_replica2'))
+# index = MeiliSearch.client.create_index(safe_index_name('City_replica2'))
```

Comment:
you can delete all this comments

Reply:
Ok, I was planning to do it for the "remove useless code" issue, but I'll do it now :)